cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -538,29 +538,62 @@ function uploadAnnotationRequest() {
                     }
                     catch(error) {
                         overlay.remove();
-                        showMessage("Parsing errors was occured. " + error);
+                        showMessage("Parsing errors was occurred. " + error);
                         return;
                     }
 
-                    const exportData = createExportContainer();
-                    exportData.create = parsed;
-
                     let asyncSave = function() {
```

Comment:
This name became unsuitable

Reply:
asyncSave is a good name here. Let's keep it.